### PR TITLE
Add new test to check the email notifications escaping

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
Just started reviewing :)

File: modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarBookingLocalServiceTest.java [hunk 1]
```diff
@@ -831,6 +833,66 @@ public void testDeleteStagingCalendarBookingDeletesLiveCalendarBooking()
...
+
+		long endTime = startTime + Time.HOUR;
+
+		long firstReminder = Time.MINUTE;
+
+		serviceContext.setWorkflowAction(WorkflowConstants.ACTION_PUBLISH);
+
+		String mailBody = "[$EVENT_TITLE$], [$TO_NAME$], [$EVENT_LOCATION$]";
+		String mailSubject = "[$EVENT_TITLE$], [$TO_NAME$], [$EVENT_LOCATION$]";
+
```
Comment (rafaprax) [inline modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarBookingLocalServiceTest.java:858]: Hey @seltong, where are you using the filters that the SubscriptionSender provides to escape context variables? please use them in the template that you are creating. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
Just started reviewing :)

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:forward

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:forward

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**